doc/Changelog.md

```diff
@@ -1,29 +1,36 @@
 Changelog {#Changelog}
 =========
 
-# master {#master}
+# Release 0.7 (09-12-2016)
+* [#353](https://github.com/BlueBrain/Livre/pull/353):
+  handle files with right extension and no schema
 * [#352](https://github.com/BlueBrain/Livre/pull/352):
-  Fix missing redraw after expose
-* [#331](https://github.com/BlueBrain/Livre/pull/331):
-  Multiple pass mode added for high quality images for data that does not fit into GPU cache
+  Fix missing redraw after expose.
+  * Fix the %300 CPU usage.
+* [#346](https://github.com/BlueBrain/Livre/pull/346):
+  Implement axis and legend for full volume bounding box.
 * [#333](https://github.com/BlueBrain/Livre/pull/333):
-  Add animation-fps parameter to limit the animation frame rate
+  Add animation-fps parameter to limit the animation frame rate.
+* [#331](https://github.com/BlueBrain/Livre/pull/331):
+  Multiple pass mode added for high quality images for data that does not fit into GPU cache.
+* [#320](https://github.com/BlueBrain/Livre/pull/3):
+  Added support for floating point volumes.
 * [#318](https://github.com/BlueBrain/Livre/pull/318):
-  * Added support for NRRD and Raw volumes
+  Added support for NRRD and Raw volumes.
 * [#317](https://github.com/BlueBrain/Livre/pull/317):
-  * Added purge support for cache
-  * The cache objects now uses RAII for allocation of data
-  * Rename TextureDataCache to DataCache, TextureDataObject to DataObject
+  Added purge support for cache.
+  * The cache objects now uses RAII for allocation of data.
+  * Rename TextureDataCache to DataCache, TextureDataObject to DataObject.
```

Comment:
What does it do for me as a user? The bullet points I definitely don't care about (as a user).